docker/observability/prometheus/rules.yml

```diff
@@ -183,3 +183,48 @@ groups:
             stolon-keeper manages Postgres configuration, along with Postgres
             reload and restarts. This alert is firing when a keeper is reporting
             a required restart that it's been unable to automatically apply.
+
+  - name: stolon-etcd.rules
+    rules:
+      - alert: StolonEtcdClientExpiryImminent
+        # We want to know if we imminently need to roll the client certificates
+        # used by stolon-pgbouncer to access etcd. This will page when a
+        # certificate will expire within 3 days.
+        expr: >
+            count by (app, backend, environment)((stolon_store_certificate_expiry_seconds - time()) < (60 * 60 * 24 * 3))
```

Comment:
Ideally you'd use the labels that we know are in the project for this, so anyone can use these. Can we rework things to use `stolon_cluster_identifier`?